apps/web/modules/integrations/webhooks/components/webhook-detail-modal.tsx

```
@@ -31,16 +41,45 @@ export const WebhookModal = ({ open, setOpen, webhook, surveys, isReadOnly }: We
+        <DialogBody>
+          <div className="flex h-full w-full flex-col">
+            <div className="flex w-full items-center justify-center space-x-2 border-b border-slate-200 px-6">
+              {tabs.map((tab, index) => (
+                <button
+                  key={tab.title}
+                  className={`mr-4 px-1 pb-3 focus:outline-none ${
+                    activeTab === index
+                      ? "border-brand-dark border-b-2 font-semibold text-slate-900"
+                      : "text-slate-500 hover:text-slate-700"
+                  }`}
+                  onClick={() => handleTabClick(index)}>
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)